hack/update/golang_version/update_golang_version.go

```diff
@@ -36,41 +36,42 @@ var (
+		".github/workflows/functional_verified.yml":                       workflowReplace,
+		".github/workflows/leaderboard.yml":                               workflowReplace,
+		".github/workflows/master.yml":                                    workflowReplace,
+		".github/workflows/pr.yml":                                        workflowReplace,
+		".github/workflows/sync-minikube.yml":                             workflowReplace,
+		".github/workflows/time-to-k8s-public-chart.yml":                  workflowReplace,
+		".github/workflows/time-to-k8s.yml":                               workflowReplace,
+		".github/workflows/translations.yml":                              workflowReplace,
+		".github/workflows/update-buildkit-version.yml":                   workflowReplace,
+		".github/workflows/update-calico-version.yml":                     workflowReplace,
+		".github/workflows/update-cloud-spanner-emulator-version.yml":     workflowReplace,
+		".github/workflows/update-cni-plugins-version.yml":                workflowReplace,
+		".github/workflows/update-containerd-version.yml":                 workflowReplace,
+		".github/workflows/update-cri-o-version.yml":                      workflowReplace,
+		".github/workflows/update-docker-version.yml":                     workflowReplace,
+		".github/workflows/update-docsy-version.yml":                      workflowReplace,
+		".github/workflows/update-flannel-version.yml":                    workflowReplace,
+		".github/workflows/update-gcp-auth-version.yml":                   workflowReplace,
+		".github/workflows/update-gh-version.yml":                         workflowReplace,
+		".github/workflows/update-golang-version.yml":                     workflowReplace,
+		".github/workflows/update-golint-version.yml":                     workflowReplace,
+		".github/workflows/update-gopogh-version.yml":                     workflowReplace,
+		".github/workflows/update-gotestsum-version.yml":                  workflowReplace,
+		".github/workflows/update-hide-minikube-bot-comments-version.yml": workflowReplace,
+		".github/workflows/update-hugo-version.yml":                       workflowReplace,
+		".github/workflows/update-ingress-version.yml":                    workflowReplace,
+		".github/workflows/update-inspektor-gadget-version.yml":           workflowReplace,
+		".github/workflows/update-k8s-versions.yml":                       workflowReplace,
+		".github/workflows/update-kubeadm-constants.yml":                  workflowReplace,
+		".github/workflows/update-kubernetes-versions-list.yml":           workflowReplace,
+		".github/workflows/update-metrics-server-version.yml":             workflowReplace,
+		".github/workflows/update-runc-version.yml":                       workflowReplace,
+		".github/workflows/update-ubuntu-version.yml":                     workflowReplace,
+		".github/workflows/yearly-leaderboard.yml":                        workflowReplace,
```

Comment:
I wonder can we do a regex all the yaml and yml files in the .github/workflows instead of adding them one by one?

Reply:
That's a good idea, can walk the directory and add them